### PR TITLE
Add libinput links and reword some settings

### PR DESCRIPTION
**Pull request by Daxtorim**
Clarified the self-explanatory stuff that isn't self-explanatory at all when you don't know about `libinput`. Also moved things around to group settings by keyboard / cursor / scrolling / focus-behavior.

PS: 
Regarding `follow_mouse = 3`, I tried to reword it to what I think the setting should be doing based on the old bullet point, but it doesn't do anything. `3` is just the same behavior as `0`. I don't know if this is a bug or if I just really don't understand what this setting is supposed to do. (I'm still on https://github.com/hyprwm/Hyprland/commit/c9eb0f3aabf4e6ea96aa6a20d2e2b10d19adeb36 as I follow [this COPR package](https://copr.fedorainfracloud.org/coprs/kasion/Hyprland-git/) on fedora, so I don't know if this has changed in the meantime)

### CODE REVIEW

**Comment by vaxerski** [comment]
difference between 0 and 3 is that in 0 the mouse cursor will always be passed to the active window, while in 3 it will be passed to whatever window it's on. That means, with 3, you can scroll on a different window (than your kb focused one), while on 0, you can't.

**Comment by Daxtorim** [comment]
Addressed the review comments.

---

> That means, with 3, you can scroll on a different window (than your kb focused one), while on 0, you can't.

Nope, doesn't work here. That is what 2 does, though.

**Comment by vaxerski** [comment]
odd. Means I've prolly broken it

**Comment by vaxerski** [comment]
fixed.

**Comment by Daxtorim** [comment]
> fixed

Can confirm.

Lemmino if you want more changes to this :eyes:

**Comment by vaxerski** [comment]
> Lemmino if you want more changes to this :eyes:

is that a motherfucking lemmino reference

good youtuber

**Comment by vaxerski** [comment]
1. thanks!
